gapic/api/artman_cloudtrace.yaml

```diff
@@ -0,0 +1,18 @@
+cloudtrace_common:
```

Comment:
Since I've committed #22, these keys would be 'common', 'python', and 'ruby' -- no 'cloudtrace_' prefix is necessary.